test/integration/arbitrum-mainnet/Loans.fork.t.sol

```diff
@@ -142,6 +141,10 @@ contract LoansForkTest is LoansTestBase {
     // whale address
     address whale;
 
+    // durations to use
+    uint durationToUse;
```

Comment:
I find the `ToUse` suffix redundant. It doesn't sound like it adds any information, but makes everything longer. Like, why else would there be a variable if not "to use", but at the same time it adds nothing about the usage itself. I'd just drop it, unless it shadows something else.